Networking/TCPSocket.hh

```diff
@@ -10,6 +10,7 @@
 #include "HTTPTypes.hh"
 #include "function_ref.hh"
 #include "fleece/Fleece.hh"
+#include "sockpp/socket.h"
```

Comment:
This doesn't seem necessary since nothing else was added to this header. (I'd like to avoid including sockpp into too many source files.)